webapp/channels/src/components/threading/virtualized_thread_viewer/virtualized_thread_viewer.test.tsx

```diff
@@ -15,9 +15,6 @@ import {TestHelper} from 'utils/test_helper';
 
 import VirtualizedThreadViewer from './virtualized_thread_viewer';
 
-// Needed for apply markdown to properly work down the line
-global.ResizeObserver = require('resize-observer-polyfill');
```

Comment:
This is now in the Jest setup script, so we don't need to import it in individual files any more